### ISSUE TITLE
Can I tell MPV to retry reading the file?

### ISSUE THREAD

**Issue by HubKing** [mpv-player/mpv#11376]
I mostly run videos off SMB shares under Gnome. Gnome seems to map SMB directories under `/run/user/1000/gvfs/smb-share:...`, so I think MPV knows nothing about SMB and thinks that it is playing a local file under that location. 

The problem is that whilst MPV is running, if I put the computer to sleep and then wake it up, MPV seems to lose the connection to the file and stops after playing the buffered amount. I don't know how it works internally, but my guess is that either the file handle gets closed by GVFS and MPV does not try to reopen the file, or right after the computer awoke, there is a brief moment when the file is not readable until the network connection gets restored, and MPV thinks that the file is no longer readable and closes the file. 

In this case, is there anyway to manually tell MPV to retry/resume reading the file? Gnome's GVFS seems to map an SMB directory to always the same local path, and at this point, the network connection has been restored, so the mapped local file should be available.

**Comment by garoto** [comment]
Either via the built-in console with:

`write-watch-later-config ; loadfile "${path}"`

Or via `input.conf` with a keybind:

`CTRL+r    write-watch-later-config ; show-text "Reloading file..." ; loadfile "${path}"`

**Comment by guidocella** [comment]
You can also use `playlist-play-index current` to not remove other playlist entries.

**Comment by HubKing** [comment]
Thanks. I guess I would use the shortcut method.

> `CTRL+r write-watch-later-config ; show-text "Reloading file..." ; loadfile "${path}"`

At first I tried to type that in the console, but the console's font was very small and somehow typing that command kept printed that command in a loop and made gnome-shell became unresponsive and the screen became black. I pressed Ctrl+Alt+F2 and logged into terminal and the killed gnome-shell to be able to use the PC again. The shortcut method does not seem to have that problem.
